content/docs/guides/next-js.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Use with Next.js - Flowbite React
+description: Learn how to install Flowbite React for your Next.js project and start developing with the most popular React-based framework built by Vercel
+---
+
+## New project
+
+Add Flowbite React to a new Next.js project:
+
+### Create project
+
+Run the following command to create a new Next.js project using `tailwind`:
+
+```bash
+npx create-next-app@latest --tailwind
+```
+
+### Install Flowbite React
+
+Run the following command to install `flowbite` and `flowbite-react`:
+
+```bash
+npm i flowbite flowbite-react
+```
+
+### Configure Tailwind CSS
+
+Open `tailwind.config.js` and update `content` and `plugins` to include Flowbite React:
+
+```js {5,9}
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: [
+    // ...
+    'node_modules/flowbite-react/lib/esm/**/*.js',
+  ],
+  plugins: [
+    // ...
+    require('flowbite/plugin'),
+  ],
+};
+```
+
+## Existing project
+
+Add Flowbite React to an existing Next.js project:
+
+### Setup Tailwind CSS
+
+1. Install `tailwindcss` and its peer dependencies:
+
+```bash
+npm i -D tailwindcss postcss autoprefixer
+```
+
+2. Generate `tailwind.config.js` and `postcss.config.js` files:
+
+```bash
+npx tailwindcss init -p
+```
+
+3. Add the paths to all of your template files in your `tailwind.config.js` file:
+
+```js {4-9}
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    './app/**/*.{js,ts,jsx,tsx,mdx}',
+    './pages/**/*.{js,ts,jsx,tsx,mdx}',
```

Comment:
You'll want to make a comment callout here differentiating Next 13 and <= 12. Like the one for the `src` directory, but

- If using >= 13 Next App Router, `app`
- If using <= 12 Next Pages Router, `pages` and `components`

Reply:
This is coming from the official documentation of tailwind (see https://tailwindcss.com/docs/guides/nextjs), I'd rather keep all of the cases at once, or I can separate them into 2 code blocks (1 for app router and 1 pages router), but I think that might bloat the content with more and more instructions.

Keeping it as vanilla as possible was the goal here (with official TW docs)

Reply:
Also, we can take into account that some big scale projects are on the process of migrating their app from `pages` router to `app` router and they have both enabled until migration is complete, therefore this setup caters for both cases.

Reply:
Good callout, that makes sense!